finetune/target_models/masked_language_model.py

```diff
@@ -29,9 +29,9 @@ def feed_shape_type_def(self):
                 {
                     "tokens": TS([None, 2]),
                     "mask": TS([None]),
-                    "mlm_weights": TS([self.config.max_masked_tokens * self.config.batch_size]),
```

Comment:
This is the key original issue. We previously were padding out to [Batch, max_masked * batch] tokens